src/index.js

```diff
@@ -22,7 +22,13 @@ const displayResult = ({ display, actions }, result) => {
     title: result.package.name,
     subtitle: result.package.description,
     clipboard: `npm i -S ${result.package.name}`,
-    onSelect: () => actions.open(result.package.links.npm)
+    onSelect: (event) => {
+      const { npm, repository } = result.package.links;
+      const url = event.altKey && repository
+        ? `${repository}#readme`
```

Comment:
Having just tried this I feel like I'd prefer this to not jump to the readme. The way I'm seeing this is that NPM already shows the readme and people would likely want the repo for things like PRs and issues. What are your thoughts on this?